gensim/models/keyedvectors.py

```diff
@@ -850,6 +851,113 @@ def n_similarity(self, ws1, ws2):
         v2 = [self[word] for word in ws2]
         return dot(matutils.unitvec(array(v1).mean(axis=0)), matutils.unitvec(array(v2).mean(axis=0)))
 
+    @staticmethod
+    def log_evaluate_word_analogies(section):
```

Comment:
Maybe better hide this method (with `_`)?

Reply:
What exactly do you mean? Or may be you can point to some example of such hiding in the existing Gensim code?

Reply:
I mean why not `_log_evaluate_word_analogies`? I asking because this method looks like a helper for `evaluate_word_analogies`, not more.

Reply:
Done.